static/css/s2.css

```diff
@@ -13349,6 +13350,38 @@ span.ref-link-color-3 {color: blue}
   color: #666666;
 }
 
+.languageHeader {
+  padding-right: 10px;
+  padding-left: 10px;
+  padding-top: 4px;
+  padding-bottom: 2px;
+  font-size: 12px;
+  font-weight: 400;
+  line-height: 18px;
+  text-align: left;
+  color: var(--medium-grey);
+
+}
+
+.languageToggleFlexContainer{
+  display: flex;
+  flex-direction: row;
+  /* padding: 10px 0px; */
+  align-items: baseline;
+  line-height: 18px;
+  height: 23px;
+  padding-bottom: 15px;
+}
+
+.languageDot{
+  font-size: 25px;
+  padding: 10px 2px;
+}
+
+.englishLanguageButton::after {
```

Comment:
I think you can actually make this a bit more general, not needing to create a special class for the enlish link

Reply:
Good thinking! 

I did this because of annoying spacing issues with r/l padding on English vs Hebrew buttons. Is that OK in this case? Happy to show you if it's clearer that way :)